extensions-core/stats/src/main/java/org/apache/druid/query/aggregation/variance/sql/BaseVarianceSqlAggregator.java

```diff
@@ -137,21 +149,77 @@ public Aggregation toDruidAggregation(
     );
   }
 
+  private static class VarPopSqlAggFunction extends SqlAggFunction
+  {
+    VarPopSqlAggFunction()
+    {
+      super(
+          SqlKind.VAR_POP.name(),
+          null,
+          SqlKind.VAR_POP,
+          ReturnTypes.AVG_AGG_FUNCTION,
+          null,
+          OperandTypes.ANY, // Can be more specific after https://github.com/apache/druid/pull/14195 is merged
```

Comment:
does this mean instead of `OperandTypes.ANY` you would use `OperandTypes.or` to check for either numeric or `RowSignatures.complexTypeChecker` with the appropriate native complex type? Any reason not to pull that change into this PR? it seems generally useful for validating any complex input types

Reply:
oh i should have read the PR description, you covered this 😛

Reply:
I pulled in the changes for `RowSignatures.complexTypeChecker`